main.js

```diff
@@ -0,0 +1,61 @@
+const booksContainerEl = document.querySelector('.books-container');
+const titleEl = document.querySelector('#title');
+const authorEl = document.querySelector('#author');
+const btnAddEl = document.querySelector('.btn-submit');
+
+if (!localStorage.getItem('booksInfo')) {
+  localStorage.setItem('booksInfo', JSON.stringify([]));
+}
+const books = JSON.parse(localStorage.getItem('booksInfo'));
+
+function uploadToStorage(books) {
+  localStorage.setItem('booksInfo', JSON.stringify(books));
+}
+
+/* eslint-disable no-use-before-define */
+function loadFromStorage() {
+  const books = JSON.parse(localStorage.getItem('booksInfo'));
+  displayBooks(books);
+}
+loadFromStorage();
+
+function removeBook() {
+  books.splice(this, 1);
```

Comment:
- [Optional] Let's practice making use of the Javascript ES6 filter() method since it doesn't mutate the original array.

Reply:
Thank you for reviewing my project, I will make the necessary corrections as suggested.